modules/network.go

```diff
@@ -18,9 +18,35 @@ type Networker interface {
 
 // NetID is a type defining the ID of a network
 type NetID string
+// Type4 is the Node's IPv4 reachability:
+type Type4 int
 
+// Type6 is the Node's IPv6 reachability
+type Type6 int
```

Comment:
Instead of Type, I would call this Reachability or Visiblity or something more explicit. 
Specially that this live in the modules package so we have all sorts of type in there.

Reply:
Eh.. your Code, your names ;-) but make a distinction 4-6